tutorials/tutorial05_visualize.ipynb

```diff
@@ -13,71 +13,92 @@
     "collapsed": true
    },
    "source": [
-    "This tutorial describes the process of visualizing and replaying the results of Flow experiments run using RL. The process of visualizing results breaks down into two main components:\n",
+    "This tutorial describes the process of visualizing the results of Flow experiments, and of replaying them. \n",
     "\n",
-    "- reward plotting\n",
-    "\n",
-    "- policy replay\n",
-    "\n",
-    "Furthermore, visualization is different depending on whether your experiments were run using rllab or RLlib. Accordingly, this tutorial is divided in two parts (one for rllab and one for RLlib). Note that this tutorial only talks about visualization using sumo, and not other simulators like Aimsun. "
+    "**Note: This tutorial is only relevant if you use SUMO as a simulator. We currently do not support policy replay nor data collection when using Aimsun. The only exception is for reward plotting, which is independent on whether you have used SUMO or Aimsun during training.**"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
+   },
    "source": [
-    "## Visualization with rllab"
+    "## 1. What is visualization"
```

Comment:
maybe change this to something like "Visualization Components". When you say "What is visualization", I expect to see a definition.

Reply:
Good point, I changed it.